### ISSUE TITLE
Why do you compute the loss_ce when the model is pretrained?

### ISSUE THREAD

**Issue by showstarpro** [zhangq327/U-MAE#9]
Why do you compute the loss_ce when the model is pretrained? It is an unreasonable setup for self-supervised learning.
The following code is from line 64 of file "engine_pretrain.py":

loss = loss_mae + args.lamb * loss_reg + loss_ce

**Comment by zhangq327** [comment]
When calculating the CE loss in the pretraining process, the encoder is frozen and the gradient will not be propagated (line 232 in models_mae.py).  We only train a linear classifier following the encoder to evaluate the learning process and the trained classifier will not be used in the downstream tasks. In summary, the CE loss will not influence the pretrained SSL models and you can just remove this term.

Hope it helps!

**Comment by showstarpro** [comment]
Thanks for your quick answer, it has benefited me greatly.
